## 1
##### Виконав базові приклади Python у файлі test.py, який прикріпив у репозиторій
## 2
##### Результат виконання команди 'python .':
##### We are in the __main__
##### 2020-10-22 00:05:41.683976
##### linux
## 3
##### Виконуємо python . -h і появляється текст з опційними аргументами
##### -h     - допомога
##### -o OPT - ця опція є вибірковою
##### -l     - ввівши цей параметр будуть виведені логи
## 4
##### Передавши python . наступний параметр, отримуємо додаткове повідомлення від програми:
##### З консолі було передано аргумент
##### ========== >> Цей текст також має вивестись << ==========
## 5
##### Якщо ввести python . --logs, то будуть виведені усі можливі логи, які прописані всередині програми.
## 6
##### Створив власну функцію в common.py та викликав її з __main__ з допомогою імпортування модулю та
##### команди common.my_func(param)
## 7
##### Створив функцію, яка виконується з помилкою та виводить ERROR повідомлення з допомогою бібліотеки logging